docs/assemblies/creating-the-data-plane.adoc

```diff
@@ -20,6 +20,9 @@ To create and deploy a data plane, you must perform the following tasks:
 ** CBO is installed and configured for provisioning.
 ** `BareMetalHosts` registered, inspected, and have the label `app:openstack`.
 * You are logged on to a workstation that has access to the RHOCP cluster as a user with `cluster-admin` privileges.
+* Review the available fields on the dataplane CRD's, further documentation on each field available in the dataplane CRD's is
+available under the <<custom-resources,Custom Resources>> documentation section.
+
 
```

Comment:
Hey, I think you can safely remove this as a prereq, because we point the user to that section in the procedure - see step 5 in https://openstack-k8s-operators.github.io/dataplane-operator/#proc_creating-a-set-of-data-plane-nodes_dataplane. 

I also think you can delete https://openstack-k8s-operators.github.io/dataplane-operator/#ref_OpenStackDataPlaneNodeSet-CR-properties_dataplane in favor of your more comprehensive resource `custom_resources.adoc`, and link to it from step 5 instead.